functions/src/utils/parsers.ts

```diff
@@ -0,0 +1,22 @@
+import { format } from "date-fns";
+
+export const EMPTY_VALUE = "-";
```

Comment:
How does it work? Why a hyphen is used? Is it a magic value?

Reply:
Hyphen is the default. It can be a blank space instead or anything.

Reply:
Do I understand correctly that there will be a hyphen in the spreadsheet if the value is empty in the database?